content/20.treatments.md

```diff
@@ -846,6 +846,23 @@ They may also engage the host immune response, encouraging the immune system to
 Upcoming work may explore the specificity of nAbs for their target, mechanisms by which the nAbs impede the virus, and improvements to antibody structure that may enhance the ability of the antibody to block viral activity.
 
+There is also some research into possible potential therapeutics or prophylactics which interact with components of the innate immune response.
+Broadly, there are a variety of toll-like receptors which are examples of pattern recognition receptors (PRRs), innate immune response components which recognize pathogen-associated molecular patterns (PAMPs) and damage-associated molecular patterns (DAMPs).
+Note that Toll-like receptors form a part of innate immune recognition and can more generally contribute to promoting both innate and adaptive responses.
+Due to the relationship between coronaviral infections, ARDS, and the toll-like receptors TLR3 and TLR4, TLR agonists and antagonists hold some potential for broad-spectrum prophylaxis.
+In mouse models, poly(I:C) and lipopolysaccharide, which are agonists of TLR3 and TLR4, respectively, showed protective effects when administered prior to SARS-CoV exposure [@doi:10.1128/JVI.01410-12].
+Another type of approach which is being investigated is the possible use of what is termed trained immunity, in particular as elicited by non-SARS-CoV-2 whole-microorganism vaccines (or other microbial stimuli), as a tool which might be shown to generate heterologous protective effects with respect to SARS-CoV-2 susceptibility or severity [@doi:10.1016/j.cell.2020.04.042].
+In a recent review [@doi:10.1038/s41577-020-0285-6], trained immunity was defined as forms of memory which are temporary (e.g., months or years, and reversible), displayed by innate immune cells and innate immune features of other cells, displayed as increased responsiveness to future same or heterologous pathogen infection or sometimes decreased responsiveness or immunological tolerance, and established through epigenetic and metabolic mechanisms; the review also notes various ways in which trained immunity may be related to pathology, inflammatory-related diseases, conditions and immunology associated with aging, and cancer.
+One type of stimulus which research indicates can induce trained immunity is BCG vaccination (BCG is an acronym for bacillus Calmette-Guerin).
+BCG is a live attenuated strain derived from an isolate of Mycobacterium bovis with the common reason for application being prevention of tuberculosis in humans.
+With respect to SARS-CoV-2 specifically, clinical trials in non-SARS-CoV-2-infected adults have been setup to assess the possible efficacy of BCG vaccination.
+Some trials enroll healthcare workers, other trials hospitalized elderly adults without immunosuppression who get vaccinated with placebo or BCG at hospital discharge, and yet another set of trials older adults (>50 years) under chronic care for conditions like hypertension and diabetes.
```

Comment:
@rando2 (side-note: I didn't realize before that I had access to change the files; recently I saw how to do it, so i can do more of the editing steps now rather than asking you to do more in addition to everything else you do!)
I used present tense as a shortcut way to cover trials at different steps (those that enrolled or enrolling, those that started or still going or will start, etc. -- it's also a bit different than the therapeutic trials).
Reminder for me: I can still/will incorporate your suggestion to start the last comma clause with "and still others" instead of "and yet another set of trials"

Reply:
Grammar is so hard in these contexts! My gut likes "are enrolling" but I think you're right that it implies they are still in the enrollment phase more than "enroll" does!